src/components/BookState.jsx

```diff
@@ -16,22 +16,25 @@ const BookState = ({
         <p>{author}</p>
         <ul>
           <li>
-            <button type="button">Comments</button>
+            <button className={styles.line} type="button">Comments</button>
           </li>
           <li>
-            <button type="button" onClick={() => dispatch(deleteBook(id))}>
+            <button className={styles.line} type="button" onClick={() => dispatch(deleteBook(id))}>
               Remove
             </button>
           </li>
           <li>
-            <button type="button">Edit</button>
+            <button className={styles.button} type="button">Edit</button>
           </li>
         </ul>
       </div>
       <div className={styles.progress}>
-        <div>
-          <h2>80 %</h2>
-          <p>completed</p>
+        <div className={styles.progressDiv}>
+          <div className={styles.progressBar} />
+          <div>
+            <h2>80 %</h2>
+            <p>completed</p>
+          </div>
         </div>
         <div className={styles.updateProgress}>
           <h3>CURRENT CHAPTER</h3>
```

Comment:
- [ ] Kindly 😃 try to increase the padding of this button here as you see here the padding of the left and right is less than the original design, so let's increase the padding to stick to the original design 😊💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
|  ![image](https://github.com/tumainimaganiko/react-bookstore/assets/83360636/ea07d367-c0c2-4e94-8bff-61b680e4a0fc)  |    ![image](https://github.com/tumainimaganiko/react-bookstore/assets/83360636/d4efaebf-9a2a-4ec5-9aa9-212ffbb917f3)  |

Reply:
Thanks for noticing that, I've already fixed the width issue.